traffic_ops/traffic_ops_golang/README.md

```diff
@@ -79,6 +79,14 @@ Go is a compiled language so any local changes will require you to CTRL-C the co
 
 `go build && ./traffic_ops_golang -cfg $HOME/cdn.conf -dbcfg ../app/conf/development/database.conf`
 
+To cross compile the code in windows to generate binaryt to run on linux, Please replace GOARCH for architecture to build
+
+`env GOOS=linux GOARCH=arm64 go build `
+
+Once binary generated, can be copied into linux machine to run
```

Comment:
Linux is a proper noun, needs capitalization.

Grammar: "Once <ins>the</ins> binary <ins>is</ins> generated, <ins>it</ins> can be copied <del>into</del><ins>onto</ins> <ins>a</ins> <del>l</del><ins>L</ins>inux machine to <ins>be</ins> run<ins>.</ins>" (note ending punctuation).